modules/plugin/jdbc/jdbc-hana/src/main/java/org/geotools/data/hana/HanaDialect.java

```diff
@@ -176,6 +186,11 @@ public void initializeConnection(Connection cx) throws SQLException {
             if ((hanaVersion.getVersion() == 1) && (hanaVersion.getRevision() < 120)) {
                 throw new SQLException("Only HANA 2 and HANA 1 SPS 12 and later are supported");
             }
+
+            if (hanaVersion.getVersion() >= 4)
```

Comment:
This check does not seem to be necessary.

Reply:
Done